.ci/azure-pipelines/build-ubuntu-16-04.yaml

```diff
@@ -12,13 +12,12 @@ jobs:
     container: env1604
     variables:
       BUILD_DIR: '$(Agent.BuildDirectory)/build'
-      CMAKE_CXX_FLAGS: '-Wall -Wextra -O2'
     steps:
       - script: |
           mkdir $BUILD_DIR && cd $BUILD_DIR
           cmake $(Build.SourcesDirectory) \
-            -DCMAKE_CXX_FLAGS="$CMAKE_CXX_FLAGS" \
             -DPCL_ONLY_CORE_POINT_TYPES=ON \
+            -DPCL_WARNINGS_ARE_ERROS=ON \
```

Comment:
ERRORS instead of ERRORS. The error is only here, which means the flag was not activated for this build.